src/VisualizationObject.ts

```diff
@@ -184,7 +184,14 @@ export namespace VisualizationObject {
         periodsAgo: number;
     }
 
-    export interface IVisualizationObject {
+    export interface IObjectWithProperties {
+        content: {
+            properties?: string;
+            references?: VisualizationObject.IReferenceItems;
+            [keys: string]: any;
```

Comment:
`keys: string]: any;` is strange here. Do we really need it?

Reply:
Yes there are serval other items AD very specific

Reply:
done

Reply:
reverted back